pkg/service/service.go

```diff
@@ -851,14 +852,23 @@ func (s *Service) restoreBackendsLocked() error {
 			logfields.L3n4Addr:  b.L3n4Addr.String(),
 		}).Debug("Restoring backend")
 		if err := RestoreBackendID(b.L3n4Addr, b.ID); err != nil {
-			return fmt.Errorf("Unable to restore backend ID %d for %q: %s",
-				b.ID, b.L3n4Addr, err)
+			log.WithFields(logrus.Fields{
+				logfields.BackendID: b.ID,
+				logfields.L3n4Addr:  b.L3n4Addr,
+			}).Error("Unable to restore backend: %s", err)
+			failed++
+			continue
 		}
-
+		restored++
 		hash := b.L3n4Addr.Hash()
 		s.backendByHash[hash] = b
 	}
 
+	log.WithFields(logrus.Fields{
+		"restored": restored,
```

Comment:
Please use `pkg/logging/logfields/` for `"restored"` and `"failed"`. If there is no suitable logfield, then you can introduce it.

Reply:
Added the new variable please take look.